recipe/meta.yaml

```diff
@@ -1,14 +1,15 @@
 {% set name = "pluggy" %}
-{% set version = "0.4.0" %}
-{% set sha256 = "dd841b5d290b252cf645f75f3bd37ceecfa0f36394ab313e4f785fe68a4081a4" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "159cc783e056c07da6552aa5aef6b1e6c0064b4f18bd49c531fd2d40aafb0ea3" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.zip
-  url: https://pypi.io/packages/source/{{ name [0] }}/{{ name }}/{{ name }}-{{ version }}.zip
```

Comment:
Would just try changing `https://pypi.io/` to `https://pypi.python.org/` and otherwise try using this URL unchanged.